CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+-   `pathExpressionEngine` and `contextRoot` now available within event handlers
```

Comment:
It was a property of the match - which is now there via extension. Doesn't matter anyway, as it was kinda useless without a root context!